src/ngraph/op/group_convolution.hpp

```diff
@@ -0,0 +1,196 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v1
+        {
+            /// \brief Batched convolution operation, with optional window dilation and stride.
+            class GroupConvolution : public Op
+            {
+            public:
+                NGRAPH_API
+                static constexpr NodeTypeInfo type_info{"GroupConvolution", 1};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                /// \brief Constructs a batched convolution operation.
+                GroupConvolution() = default;
+                /// \brief Constructs a batched convolution operation.
+                ///
+                /// \param data_batch The node producing the input data batch tensor.<br>
+                /// `[N, C_IN, D1, ... Df]`
+                /// \param filters The node producing the filters tensor.<br>
+                /// `[C_OUT, C_IN, F1, ... Ff]`
+                /// \param strides The strides.<br>
+                /// `[f]`
+                /// \param dilations The dilations.<br>
+                /// `[f]`
+                /// \param pads_begin The beginning of padding shape.<br>
+                /// `[f]`
+                /// \param pads_end The end of padding shape.<br>
+                /// `[f]`
+                /// \param auto_pad The pad type for automatically computing padding sizes.<br>
+                /// `[f]`
+                ///
+                /// Output `[N, C_OUT, R1, ... Rf]`
+                ///
+                GroupConvolution(const Output<Node>& data_batch,
+                                 const Output<Node>& filters,
+                                 const Strides& strides,
+                                 const CoordinateDiff& pads_begin,
+                                 const CoordinateDiff& pads_end,
+                                 const Strides& dilations,
+                                 const PadType& auto_pad = PadType::EXPLICIT);
+
+                size_t get_version() const override { return 1; }
```

Comment:
Shouldn't this method be removed or return the value from `type_info` instead?

Reply:
Remove

Reply:
Done.